core/trino-main/src/main/java/io/trino/operator/join/PartitionedConsumption.java

```diff
@@ -46,7 +46,7 @@
     @Nullable
     private List<Partition<T>> partitions;
 
-    PartitionedConsumption(
+    public PartitionedConsumption(
```

Comment:
Assuming the change here and in SpilledLookupSourceHandle was necessitated by the new forked package, should they be set back to package private in some subsequent commit since the non-spilling join operator doesn't use them ?